data/net/minecraft/core/Vec3i.mapping

```diff
@@ -39,12 +40,15 @@ CLASS net/minecraft/core/Vec3i
 		ARG 1 position
 		ARG 2 useCenter
 	METHOD distSqr (Lnet/minecraft/core/Vec3i;)D
-		COMMENT Calculate squared distance to the given Vector
-		ARG 1 vector
+		COMMENT Calculate squared distance to the given Vector.
+		ARG 1 to
 	METHOD distSqr (Lnet/minecraft/core/Vec3i;Z)D
-		ARG 1 vector
+		ARG 1 to
 		ARG 2 useCenter
+	METHOD east ()Lnet/minecraft/core/Vec3i;
+		COMMENT Creates a new Vec3i one unit east.
 	METHOD east (I)Lnet/minecraft/core/Vec3i;
+		COMMENT Creates a new Vec3i east by the distance.
```

Comment:
This is technically incorrect, since if the distance is `0`, then it returns the same vector object.